packages/bsky/src/services/moderation/status.ts

```diff
@@ -0,0 +1,219 @@
+// This may require better organization but for now, just dumping functions here containing DB queries for moderation status
```

Comment:
I know we're not using functional style in most places other than util functions so may be this needs to be converted to a class to align with other services?